### ISSUE TITLE
Commitment for fx51 and Later, but compatible at fx56 up to v2.10.17-keep accuracy

### ISSUE THREAD

**Issue by ALTanir** [AdguardTeam/AdguardBrowserExtension#1450]
Please do not describe any version you release as it is compatible for firefox version specific and Later **-when if, it can install properly but settings show the blank page or empty space on menu or unable to option clicks**. Yes, adguard version-2.10.17 for fx56.0b5 works properly good. And more is- its a best blocker than another top. _When the later version can not work perfect, it seems as not compatible for fx56-win7. So the latest version is for fx57+._ I have stored-save it for future.
Another description may be the device have incompatibilities -this can confirmed by other software installations and performances. 

> For example: Water-tank have spaces to fills-up by water-but you keeps there some hand-pots of water on full of tank. Accuracy collected on many devices but no exclusions.
So my feedback is to make it such functional actions that will work all options accurate with no errors. ### That is, describe the adguard-version is on correct fx-version accurately
![git -upload](https://user-images.githubusercontent.com/10827600/61874124-6d9bbc00-af09-11e9-9a85-6be1c1ec9286.jpg)

**Comment by TheHasagi** [comment]
@ALTanir 

Hello,

Can you specify the version of your browser and adguard extension? Also, what kind of error do you facing?

**Comment by ALTanir** [comment]
@TheHasagi Thank you very much for responses. Required version fx56.0b5 for this report.
All adguard versions(at present) can install properly -for this, please do not collect the case about installation. Just the settings option, that opens a new tab-page. This page have options and listings. So I can do any changes perfectly. This only, I choose this extension is my best adblocker.
After update the version from 2.10.17 to later- I found it working just for blocking ad. When clicked toolbar button - white/blank popup'screen was showing even after restarting browser and pc. Then emailed to authority and replied as may causing older device; And they will try to solve it responsibly. Exactly that blank popup'screen was solved next time properly.
Then that new version(more later); I found more problem is- in the settings-button (in popup'screen) clicked& loaded-page(not a popup'screen) detected a Blanked tab page; Just menu button text can get viewed only(no clickable functions). Then I didn't reported it as my device may not compatible or was waiting for next more one release.
After more next version released, that new blanked tab page does not show even a option-menu text - fully blank. Same time. searching for "disabled addon_certificate expired problem" I have participated in GitHub and more some community forums. Submitted some issues, comments reports etc. So its stepped for Adguard authority by me about problem.
If authority has to say, released **later(of 2.10.17) versions for fx57+ and fx56- on v2.10.17**. That is correct communities. When you say, those blank problems but have compatibility; I have to report you for to correct, explicit explains in the finder link. That is, ### _its(later version) for fx57+ instead of 51+_
That's all, Thanx again

**Comment by ALTanir** [comment]
@TheHasagi 
**"..... confirm it with release version 3.2.1?"**
Answer: Same info for up to version 3.2.1

**Comment by ALTanir** [comment]
Strange, evenly- but not its a perfection !! Yes-today, I have restore my roaming&local>Mozilla'folder from backup of 20 days ago. Detected different report on reject my above says. That is, version 3.0.11 seems complete, special and accurate behaviors on the settings option-page; Works good- fully OK with other options and popup screens.
Also I confirm its temporarily 30% ensured. Because I have marked that all specific versions (11 files.xpi) in my mass store as which is how do? for the version _3.0.11:- renamed as with unable@settings sometime._
@TheHasagi **So if you have checked up my report-feedback- you may define it inappropriate and marked me not compatible device. Not sure? But I say- after heavy usage like gaming sites, it detects the blank page as unable for settings usage to analyze**. Causing heavy usage, I always care my pc and apps by cache cleaning and large history clearing to prevent unwanted stores. Like optimizer, protector, element blocking, defragement and many many security functional ext.&soft. Thanx to listen for unwanted more

**Comment by TheHasagi** [comment]
@ALTanir 

As it was spoken before, I need to understand that problems did not remain anymore. For more investigation, I need the results with your configuration and the latest release version. Furthermore, all supported browser versions were described in the manifest:

- Chrome: 55
- Firefox: 52
- Opera: 42
- Edge: 15

**Comment by ALTanir** [comment]
@TheHasagi 
Yes, that problem is not exist anymore. My previous comment was about version 3.0.11- its working perfectly without any problem ### since today. While the problem were detecting once, but I don't know how did it solved now- pray for keep well. On other browser I did not found such info - because, those all are latest version. Only for firefox, I need some older extensions to use. Firefox latest versions detect auto-reload problem on multiple tabs when focusing after a few minutes - I can not define it how to stop auto-reloads.
Another tips for adguard adblocker:- Last year, I have found its blocking ads but a few ads were not blocked properly. It seems ad-screen(popup) but no image or text of that ads. This -I defined software is very poor to fight. Then tried up settings to do some changes. When I switched off/on and then undo to previous position on the options about adblock and "Use optimized filters". Then it worked perfect with no error. So I think, **it happened like its starter case.** Because, In that day, I installed browser newly and so this extension seems unable to catch the line. This happens sometimes on software also while on any hardware machines. _Causes first startups, usage firstly or not a regular ways- so the machine error may occurs._ That's all about adguard extension for I like it very much. Thank you
